templates/vhosts80/rpm.conf

```diff
@@ -1,3 +1,2 @@
-Alias /pulp/repos /var/www/pub/yum/http/repos
```

Comment:
So these aliases are no longer needed for RPM content; lazy handles it differently?